widgetastic_manageiq.py

```diff
@@ -878,3 +878,144 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.inrease()
+    """
+    def __init__(self, parent, locator, logger=None):
+        super(Stepper, self).__init__(parent=parent, logger=logger)
+
+        self._stepper_control = self.browser.element(locator, parent=parent)
```

Comment:
I thought it might save some time and resources because the element was already prepered. I probably need to spend some time and bury into widgetastic internals deeper.